src/Services/src/CloudFoundry/CloudFoundryService.cs

```diff
@@ -189,7 +189,7 @@ public async Task<DetailedResult<List<CloudFoundryOrganization>>> GetOrgsForCfIn
                 }
                 else
                 {
-                    var msg = $"Something went wrong while trying to request orgs from {apiAddress}: {originalException.Message}";
+                    var msg = $"{originalException.Message}. See logs for more details.";
```

Comment:
It's good practice to use semantic log messages rather than interpolated strings for logging.

In this example, that would be:
    `_logger.Error("{Error}. See logs for more details.", originalException.Message);`

Note `{Error}` above is not a variable, it's just a semantic name. E.g., you could name it `{Message}`.  Important is to use meaningful names and be consistent.

The logging system will perform string building *if* needed.  E.g., log level has been met and there is a log destination available.

2 benefits to this approach:
1. analysis; semantic log systems can be queried.  If you use consistent semantic names, you can search logs for with something equivalent to "show me all log messages where `Error` contains 'See logs'."
1. performance; not an issue in this case, but in a backend service with lots of logging, you don't want to pay the price for building strings if, say, it's a Debug level log message and Debug messages are currently not being logged